mockOSApi.v2/Services/ProcessHandler.cs

```diff
@@ -1,66 +1,82 @@
 using System.Text.Json;
 using mockOSApi.Repository;
 using mockOSApi.Models;
+using mockOSApi.DTO;
 
 namespace mockOSApi.Services;
 
 
-public interface IProcessHandler {
-    public IEnumerable<MockProcess> AllProcesses { get; }
+public interface IProcessHandler
```

Comment:
This should be IProcessService.
C# also uses the concept of 'Handlers', but for something totally different

Reply:
renamed